flameCalculator/main.js

```diff
@@ -363,21 +369,41 @@ function numberOfLines(main_tier, secondary_tier, combo_one_tier, combo_two_tier
 function getTierProbability(main_tier, secondary_tier, combo_one_tier, combo_two_tier, combo_three_tier, combo_four_tier, combo_five_tier, all_stat_tier, attack_tier, flame_type, boss_tier, dmg_tier, non_advantaged_item, main2_tier, main3_tier, combo_six_tier) {
     var list = [main_tier, secondary_tier, combo_one_tier, combo_two_tier, combo_three_tier, combo_four_tier, combo_five_tier, all_stat_tier, attack_tier, boss_tier, dmg_tier, main2_tier, main3_tier, combo_six_tier]
     var index = 0
-    var probability = 0
+    var probability = 1
     var tier_probabilities = getTierProbabilities(flame_type, non_advantaged_item);
     while (index < list.length) {
         var tier = list[index];
         if (tier > 0) {
-            if (probability == 0) probability = tier_probabilities[tier]
-            else {
-                probability = probability * tier_probabilities[tier]
-            }
+            probability = probability * tier_probabilities[tier]
```

Comment:
This is where I fixed that bug. If the tier probability for a tier was 0 then it would correctly set the probability to 0. But then a later tier that wasn't 0 would trigger the `probability == 0` condition and set the probability to that chance.

Changing the starting probability to 1 fixes this issue since we don't need to set the probability to the first possible tier any more. Since we multiply regardless, any options that can't happen still come out as 0.